src/main/java/org/jabref/gui/fieldeditors/FieldNameLabel.java

```diff
@@ -4,15 +4,67 @@
 import javafx.geometry.Pos;
 import javafx.scene.control.Label;
 
+import javafx.scene.control.Tooltip;
 import org.jabref.model.entry.field.Field;
+import org.jabref.model.entry.field.StandardField;
 
 public class FieldNameLabel extends Label {
+    private Tooltip tip;
+    private static final String AUTHOR = "author";
+    private static final String BIBTEXKEY = "bibtexkey";
+    private static final String JOURNAL = "journal";
+    private static final String TITLE = "title";
+    private static final String YEAR = "year";
 
-    public FieldNameLabel(Field field) {
+    private static final String AUTHOR_TIP = "Multiple authors separated with 'and', e.g. author1 and author2";
+    private static final String BIBTEXKEY_TIP = "[First author'last name][Article year] e.g. Jones2020";
+    private static final String JOURNAL_TIP = "The name of the journal";
+    private static final String TITLE_TIP = "The title of the article";
+    private static final String YEAR_TIP = "The year of publication, e.g. 2005";
+
+    public FieldNameLabel(Field field, String name) {
         super(field.getDisplayName());
 
         setPadding(new Insets(4, 0, 0, 0));
         setAlignment(Pos.CENTER);
         setPrefHeight(Double.POSITIVE_INFINITY);
+        setTip(name);
+    }
+
+    public void setTip(String name){
```

Comment:
In case the tooltip is included int he `StandardField`, this whole mapping becomes obsolete.

Reply:
I wouldn't put it in `StandardField` as this class is a logic/model class while the tooltip is a GUI-related thing. Thus, having such a mapping in the gui is fine.